tox.ini

```diff
@@ -57,6 +57,9 @@ deps =
     jwst: jwst[test] @ git+https://github.com/spacetelescope/jwst.git
     romancal: romancal[test] @ git+https://github.com/spacetelescope/romancal.git
     oldestdeps: minimum_dependencies
+    devdeps: astropy>=0.0.dev0
+    devdeps: numpy>=0.0.dev0
+    devdeps: scipy>=0.0.dev0
```

Comment:
Same as all by previous comments, see https://github.com/spacetelescope/romancal/pull/695#discussion_r1189078751 and https://github.com/spacetelescope/romancal/pull/695/files/b791170675848ecb84fc1068587528c22403b284#r1189076034